test/t/unit/test_unit_get_first_arg.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+
+from conftest import assert_bash_exec
+
+
+@pytest.mark.bashcomp(cmd=None)
+class TestUnitGetFirstArg:
+    @pytest.fixture(scope="class")
+    def functions(self, bash):
+        assert_bash_exec(
+            bash,
+            '_comp__test_unit() { local -a "words=$1"; local cword=$2 ret=; shift 2; _comp_get_first_arg "$@" && printf "%s\\n" "$ret"; return 0; }',
+        )
+
+    def test_1(self, bash, functions):
+        assert_bash_exec(bash, "_comp__test_unit '()' 0")
+
+    def test_2(self, bash, functions):
+        """a b| should set args to 1"""
```

Comment:
Most docstrings throughout this file seem to be off, as we're returning the first arg (if any), not the number of args.

Reply:
Thank you. You are right. Those tests are just converted from the ones for `_count_args`, but I forgot to update the docstrings.